lib/proxy_api/proxy_exception.rb

```diff
@@ -8,7 +8,15 @@ def initialize(url, exception, message, *params)
     end
 
     def message
-      super + ' ' + _('for proxy') + ' ' + url
+      super + proxy_exception_response(exception) +
+        ' ' + _('for proxy') + ' ' + url
+    end
+
+    private
+
+    def proxy_exception_response(exception)
+      return '' unless exception.respond_to?(:response)
```

Comment:
I think `exception` here is either the same or a new instance of _this_ exception. I think this needs to be `wrapped_exception`.